nRF9160Master/src/System/SystemHandler.c

```diff
@@ -158,6 +169,8 @@ void ProcessDeviceState()
                     printk("INFO: Connected to WiFi\n\r");
                     StarTimerTask(30);
                     SetDeviceState(WIFI_DEVICE);
+                    WifiStatusFlag = true;
+                    SetWifiStatus(WifiStatusFlag);
```

Comment:
Could you please directly pass the boolean as _SetWifiStatus(true);_ instead of using an extra variable here.